packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.service.js

```diff
@@ -101,11 +101,12 @@ export default class PrivateDatabaseOrderCloudDb {
     }));
   }
 
-  static getOrderableDatacenters(plans) {
+  static getOrderableDatacenter(plans) {
+    // there is only one available datacenter per geographical zone
     return PrivateDatabaseOrderCloudDb.filterOrderableItems(
       plans,
       DATACENTER_CONFIGURATION_KEY,
-    );
+    )[0];
```

Comment:
Why the first one ?

Reply:
```
// there is only one available datacenter per geographical zone
```
Are we sure about that @lucas-chaigne ?

Reply:
Hello,
yes, it is a request coming from db team (https://projects.dsi.ovh/browse/PUDBLOW-1360)
"Datacenter : Useless since we only have one datacenter and no more datacenter is planned."
I checked with Mikael Davranche and @wdhif and there is one dc in Europe, one in Canada.